### ISSUE TITLE
USER ISSUE: Server drops all players

### ISSUE THREAD

**Issue by RebelWisp** [StrangeLoopGames/EcoIssues#11052]
**Version:** 0.8.0.0 beta staging-1785734b-2

**Steps to Reproduce:**
1. Nothing seems to make it worse or better. It just happens every 20 minutes sometimes. Occassionally there is up to a 5 minute roll back.

**Expected behavior:**
continued to play.

**Actual behavior:**
The following error message popped up:
```
Connection timed out
``

No actual bug, just disconnects`

**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up:
```
Connection timed out
``

No actual bug, just disconnects`

**Comment by metachronism** [comment]
probably world layer tick high CPU usage - what world size was being used?

**Comment by mirasrael** [comment]
Hi @RebelWisp! 

Thanks for report. Few questions for troubleshooting.

How do you run server?
- from client?
- as dedicated server?
Can you attach logs from your server's Dump directory?
Can you attach your client logs? `AppData/LocalLow/StrangeLoopGames/output_log.txt`
Can you attach your server save? In server's directory: `Storage/Game.eco` for dedicated server or `Storage/WhatEverYourSaveName.eco` for client's run.

**Comment by mirasrael** [comment]
@RebelWisp thanks for update. Was it related to any in-game events or completely random? 
Unfortunately it is hard to diagnose such problems without logs, because we have another servers running 2-4km world without problems.
So have to wait until we get more information.

**Comment by mirasrael** [comment]
Probably fixed in 8.0.1. Re-open if still a case.
